models/todos.js

```diff
@@ -0,0 +1,24 @@
+module.exports = (sequelize, DataTypes) => {
+  const Todo = sequelize.define('Todo', {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true
+    },
+    created_at: {
+      type: DataTypes.DATE,
+      allowNull: false
+    },
+    updated_at: {
+      type: DataTypes.DATE,
+      allowNull: false
+    }
+  }, {
+    tableName: 'todos',
+    timestamps: true,
+    updatedAt: 'updated_at',
+    createdAt: 'created_at'
+  });
+
+  return Todo;
+};
```

Comment:
Ensure to add a newline at the end of the file to adhere to coding standards and improve compatibility with various tools.